### PR TITLE
RIP-36 initial PR with auth and user controllers

### PR DESCRIPTION
**Pull request by mlincoln**
https://jira-secure.berkeley.edu/browse/RIP-36

test coverage to come

### CODE REVIEW

File: ripley/api/auth_controller.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
+"""
+Copyright ©2022. The Regents of the University of California (Regents). All Rights Reserved.
```
Comment (lyttam) [inline ripley/api/auth_controller.py]: @mlincoln Copyright year should be 2023 on all files.

Comment (johncrossman) [reply ripley/api/auth_controller.py]: @lyttam @mlincoln - I created https://jira-secure.berkeley.edu/browse/RIP-58 to update copyright year. All other tools need the same treatment, for which I created more Jiras. 😄

**Comment by johncrossman** [comment]
@mlincoln - You will need to resolve conflicts (per above). It is fine to use Github's in-browser editor to fix `default.py`.

File: ripley/api/auth_controller.py (hunk 1 above)
Comment (lyttam) [reply ripley/api/auth_controller.py]: Thanks @johncrossman!

**Comment by mlincoln** [comment]
I changed 2022 -> 2023. Thanks for the tip on the in-browser editor @johncrossman.

**Comment by pauline2k** [comment]
Thanks for changing the dates, @mlincoln - I think the only thing left to do is resolve the conflict in config/default.py (for which the in-browser editor should also work).

**Comment by johncrossman** [comment]
@mlincoln - Next time we meet, we can discuss rebasing to resolve conflicts. For now, I'll make the edit to resolve this conflict. This is good work and I'd like to build on it.

**Pull request merged**